src/BugsnagUnity/Client.cs

```diff
@@ -138,7 +138,7 @@ void Notify(string condition, string stackTrace, LogType logType)
           Notify(new Exception[]{exception}, exception.HandledState, null, logType);
         }
       }
-      else
+      else if (logType.IsGreaterThanOrEqualTo(Configuration.BreadcrumbLogLevel))
```

Comment:
This is inconsistent with the notifier spec. If `LogType.Log` is specified in `config.BreadcrumbLogLevel`, then log-level messages will not be recorded as breadcrumbs - whereas the spec states they should be.